php/Controllers/SettingsController.php

```diff
@@ -166,122 +319,247 @@ private function getProfile(): ?ProfileModel {
         return $profile;
     }
 
+    /**
+     * WP Admin main settings page
+     * @void string HTML for page holding form with setting inputs
+     */
     public function settings_page_cb() {
-        echo sprintf("<div class=\"wrap\">
-                <h2>%s</h2>
-                <form method=\"post\" action=\"options.php\">
-        ", $this->page_title);
 
-        settings_fields(self::page_slug);
-        do_settings_sections(self::page_slug);
+        ?>
+        <div class="wrap">
+
```

Comment:
Is the excessive use of empty newlines a WordPress plugin convention?

Reply:
Compacted the file (removed whitelines where possible)